xarray/core/coordinates.py

```diff
@@ -796,6 +797,126 @@ def _ipython_key_completions_(self):
         ]
 
 
+class DataTreeCoordinates(DatasetCoordinates):
```

Comment:
Can you just subclass from `Coordinates` instead of `DatasetCoordinates`?

Reply:
done in https://github.com/pydata/xarray/pull/9451/commits/1bf50829d53ba1a30f91963a2499ce77829409e7